src/game.js

```diff
@@ -0,0 +1,74 @@
+var Game = {
```

Comment:
The code above is from the bundle.js file that gets compiled by web pack. I added it to the gitignore so it shouldn't show up anymore after this.